Resources/doc/index.rst

```diff
@@ -50,7 +50,7 @@ Require the bundle with composer:
 
 .. code-block:: bash
 
-    $ composer require friendsofsymfony/user-bundle "~2.0.0-beta2"
+    $ composer require friendsofsymfony/user-bundle "~2.0.0"
```

Comment:
use either ``~2.0`` or ``^2.0``. The current constraint forbids to upgrade to 2.1, and we don't want that